manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "country-data-settings",
-  "version": "0.3.0",
```

Comment:
why beta?

Reply:
because I wanted to test in a production workspace